apps/console/src/features/applications/components/help-panel/overview.tsx

```diff
@@ -103,7 +103,7 @@ export const HelpPanelOverview: FunctionComponent<HelpPanelOverviewPropsInterfac
             {
                 !isOIDCConfigsLoading ? (
                     <Grid>
-                        {/*  {
+                        { /*  {
```

Comment:
Why do we need this line. Shall we check and add a TODO if we are going to re- enable it near future

Reply:
Fixed with commit 2db59e4